### PR TITLE
Reducers

### PR DESCRIPTION
**Pull request by c00p75**
- [x] Add Redux Toolkit.
- [x] Structure your application files using a "feature folder" approach and use the [ducks pattern](https://github.com/erikras/ducks-modular-redux).
- [x] Write books and categories actions and reducer.
- [x] Configure the Redux Store.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// Initializing constatnt action variables
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
```
Comment (TedLivist) [inline src/redux/books/books.js]: - [x] The values of the action types should follow the ducks' pattern.

  ```
  const ADD_BOOK = 'bookstore/books/ADD_BOOK';
  const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
  ```

File: src/redux/books/books.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+const defaultState = [];
+
+export const booksReducer = (state = defaultState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.book];
+    default:
+      return state;
+  }
+};
```
Comment (TedLivist) [inline src/redux/books/books.js]: - [x] Kindly implement the switch action for when the action type is `REMOVE_BOOK`. Kindly filter the books using the `Array.filter()` method

File: src/redux/categories/categories.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```
Comment (TedLivist) [inline src/redux/categories/categories.js]: - [x] Kindly use the duck's pattern for the action type.

  ```
  const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS'
  ```

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @c00p75,

Good job so far!
✔️ Implementation of ducks pattern for file arrangement
✔️ Use of the redux toolkit to configure the store
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @c00p75👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**